src/aapns/models.py

```diff
@@ -97,6 +97,10 @@ class PushType(Enum):
 
     alert = "alert"
     background = "background"
+    voip = "voip"
```

Comment:
🤔 I recall that `void` notifications have different requirements, for example the size limit is larger... I wonder what other requirements are there and a `void` notification would actually be functional with this library without other changes 🤔

Reply:
s/void/voip/g

Reply:
aapns doesn't actually validate payload size, so this change should be fine.

Reply:
https://github.com/HENNGE/aapns/blob/80daee4fb7ceb7df6527c891c8cd964c2d16be0a/src/aapns/connection.py#L24-L27